src/sagas/range-saga.js

```diff
@@ -832,6 +847,15 @@ function* multiDelete({ payload: { ranges } }) {
   }
 }
 
+function* clearCustomColorsSaga() {
+  const rangeList = yield select(getRangeList);
+  const rangeIds = Object.keys(rangeList);
+
+  for (let rangeId of rangeIds) {
+    yield put(unsetRangeColor(rangeId));
```

Comment:
Had some previous comments on the PR about undo-ing this, I don't think though that any dramatic changes would be needed if we wanted go back and implement the undo-redo. Having the operations granular is exactly whats required 👍